Utilities/src/test/java/edu/stanford/hivdb/mutations/MutationTest.java

```diff
@@ -428,6 +436,7 @@ public void testGetHumanFormat() {
 
 	@Test
 	public void testParseString() {
+		Mutation.parseString(Gene.RT, "T69Insertion");
```

Comment:
No assertion?

Reply:
I actually meant to remove this line, but I think it's a good test case to include. So I've asserted it.